bin/templates/project/__PROJECT_NAME__/Images.xcassets/AppIcon.appiconset/Contents.json

```diff
@@ -121,6 +121,12 @@
       "scale" : "2x"
     },
     {
+      "idiom" : "ipad",
+      "size" : "83.5x83.5",
```

Comment:
I think this might be a mistake. Formatting is messed up on line 117. Lines 123-128 look like a copy of lines 117-122